src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import duke.data.exception.DukeException;
+import duke.parser.Parser;
+
+public class ParserTest {
```

Comment:
Could include JavaDoc comments for this class? Perhaps same for other test classes? 🤔 
(https://se-education.org/guides/conventions/java/intermediate.html#comments)